### PR TITLE
Restore online test suite. 

### PR DESCRIPTION
**Pull request by nabobalis**
Should restore the online tests. 
Need to decide what to do about the 32bit issue.

### CODE REVIEW

File: sunpy/database/tests/test_tables.py [hunk 1]
```diff
@@ -505,10 +505,60 @@ def test_entry_from_query_results_with_none_wave_and_default_unit(
...
+            observation_time_start=datetime(1996, 4, 11, 0, 0),
+            observation_time_end=datetime(2012, 9, 14, 0, 0),
+            instrument='VIRGO', size=512000.0, wavemin=None,
+            wavemax=None)
+        ]
 
-    for e in expected:
-        assert e in entries
+    for e in entries:
+        assert e in expected
```
Comment (nabobalis) [inline sunpy/database/tests/test_tables.py]: This order made more sense?

**Comment by Cadair** [comment]
Is the 32bit build using all the same deps as our other builds? If this really is a 32bit issue that's *proper* weird.

**Comment by nabobalis** [comment]
Yes:

```
          command: pip3 install cython numpy scipy astropy matplotlib asdf | cat
          command: pip3 install sqlalchemy scikit-image glymur drms zeep beautifulsoup4 requests python-dateutil pytest pytest-cov pytest-mock pytest-xdist hypothesis pytest-astropy pytest-rerunfailures parfive[ftp] | cat
          command: PYTHONHASHSEED=42 python3 setup.py test --parallel=4
```

**Comment by Cadair** [comment]
It's not using tox though, so I suppose there could be some differences. For a start it's doing it in two runs not one.

**Comment by nabobalis** [comment]
I should replace it with the tox env. However I suspect (looking at this 32bit thatp assed https://circleci.com/gh/sunpy/sunpy/14328?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) it could have been the update to Astropy 3.2 that broke it.

**Comment by Cadair** [comment]
@ayshih Do you have any ideas on why the horizons comparison test is failing here?

https://dev.azure.com/sunpy/sunpy/_build/results?buildId=2600&view=logs&jobId=7c6c8039-fcf8-56d3-66cb-1f8277ff167e&taskId=95cbb3cb-87f4-50d2-5d7d-86770b441620&lineStart=56&lineEnd=100&colStart=1&colEnd=92

**Comment by ayshih** [comment]
Go ahead and update the tolerance from 25 km to 35 km.

**Pull request merged**